src/main/java/edu/greenblitz/robotName/RobotConstants.java

```diff
@@ -6,9 +6,9 @@
 public class RobotConstants {
 
     public static final Robot.RobotType ROBOT_TYPE = Robot.RobotType.ROBOT_NAME;
-	public static final String SIMULATION_LOG_PATH = System.getProperty("user.home") + "\\Desktop\\SimulationLogs";
-	public static final String USB_LOG_PATH = "/media/usb/";
-	public static final String SAFE_ROBORIO_LOG_PATH = "/logs/";
+    public static final String SIMULATION_LOG_PATH = System.getProperty("user.home") + "\\Desktop\\SimulationLogs";
```

Comment:
```suggestion
    public static final String SIMULATION_LOG_PATH = String.format("%s\\Desktop\\SimulationLogs", System.getProperty("user.home"));
```

String concatenation with `+` is inefficient